.env.local

```diff
@@ -0,0 +1,2 @@
+RAZZLE_API_PATH= https://v3.io-comune.redturtle.it/api
+RAZZLE_LEGACY_TRAVERSE=true
```

Comment:
Remove this whole file. Use the `.env` name without `.local` when developing core volto.